<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <!-- 放大窗口，避免按钮被裁剪 -->
   <rect><x>0</x><y>0</y><width>680</width><height>560</height></rect>
  </property>
  <property name="minimumSize">
   <size><width>680</width><height>560</height></size>
  </property>
  <property name="windowTitle"><string>登录</string></property>

  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin"><number>20</number></property>
   <property name="topMargin"><number>20</number></property>
   <property name="rightMargin"><number>20</number></property>
   <property name="bottomMargin"><number>20</number></property>

   <!-- 只保留蓝色大标题 -->
   <item>
    <widget class="QLabel" name="lbTitle">
     <property name="objectName"><string>headline</string></property>
     <property name="text"><string>工业现场远程专家支持系统</string></property>
     <property name="alignment"><set>Qt::AlignHCenter|Qt::AlignVCenter</set></property>
     <property name="margin"><number>8</number></property>
    </widget>
   </item>

   <!-- 表单区：无边框，仅布局 -->
   <item>
    <widget class="QFrame" name="card">
     <property name="frameShape"><enum>QFrame::NoFrame</enum></property>
     <property name="frameShadow"><enum>QFrame::Plain</enum></property>
     <layout class="QVBoxLayout" name="cardLayout">
      <property name="leftMargin"><number>24</number></property>
      <property name="topMargin"><number>12</number></property>
      <property name="rightMargin"><number>24</number></property>
      <property name="bottomMargin"><number>12</number></property>

      <item><widget class="QComboBox" name="cbRole"/></item>
      <item><widget class="QLineEdit" name="leUsername">
        <property name="placeholderText"><string>账号</string></property>
      </widget></item>
      <item><widget class="QLineEdit" name="lePassword">
        <property name="placeholderText"><string>密码</string></property>
        <property name="echoMode"><enum>QLineEdit::Password</enum></property>
      </widget></item>

      <item>
       <layout class="QVBoxLayout" name="btnsLayout">
        <item><widget class="QPushButton" name="btnLogin">
          <property name="text"><string>立即登录</string></property>
        </widget></item>
        <item><widget class="QPushButton" name="btnToReg">
          <property name="text"><string>前往注册</string></property>
        </widget></item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>

   <item><spacer name="spacer_bottom">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint"><size><width>20</width><height>20</height></size></property>
   </spacer></item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
